optimizely/event_builder.py

```diff
@@ -85,6 +86,15 @@ def _get_anonymize_ip(self):
 
     return self.config.get_anonymize_ip_value()
 
+  def _get_bot_filtering(self):
+    """ Get bot filtering bool
+
+    Returns:
+      'botFiltering' value in the datafile.
```

Comment:
nit. `Boolean representing whether bot filtering is enabled or not.`

I would similarly also change the comment for IP anonymization in the method above to 

Boolean representing whether IP anonymization is enabled or not.